android/src/main/java/com/hoxfon/react/RNTwilioVoice/TwilioVoiceModule.java

```diff
@@ -225,6 +227,27 @@ public void onError(RegistrationException error, String accessToken, String fcmT
         };
     }
 
+    private UnregistrationListener unregistrationListener() {    //
+        return new UnregistrationListener() {
+            @Override
+            public void onUnregistered(String accessToken, String fcmToken) {
+                if (BuildConfig.DEBUG) {
+                    Log.d(TAG, "Successfully unregistered FCM");
+                }
+                // eventManager.sendEvent(EVENT_DEVICE_UNREGISTERED, null);
```

Comment:
```suggestion
```

Reply:
Why is it reporting not ready here and not unregistered? What is the difference?

Reply:
Not ready is correct from my side. Not registered was a proposal for you if you think we should add this state.